pkg/fission-cli/cmd/spec/spec.go

```diff
@@ -162,6 +162,78 @@ func save(data []byte, specDir string, specFile string) error {
 	return nil
 }
 
+// called from `fission * create --dry`
+func SpecDump(resource interface{}) error {
```

Comment:
Please don't duplicate `SpecSave`code directly, it lowers the maintainability of codebase.
We can instead of moving the duplicate part to an independent function and call it in `SpecSave` and `SpecDump`.